frontend/src/layout/navigation/DemoWarning.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { useValues } from 'kea'
+import { Alert } from 'antd'
+import { StarOutlined, SettingOutlined } from '@ant-design/icons'
+import { userLogic } from 'scenes/userLogic'
+import { LinkButton } from 'lib/components/LinkButton'
+import { Link } from 'lib/components/Link'
+
+export function DemoWarning(): JSX.Element | null {
+    const { user, demoOnlyUser } = useValues(userLogic)
+
+    if (!demoOnlyUser) {
```

Comment:
Can be an improvement for later, but I would keep this banner for all demo projects (regardless of whether it's only the demo one) and just change the copy. Because a) it's useful if you think in the context of Hogflix by itself and b) what happens after the user has created the project but hasn't completed the setup?

Reply:
Since the copy/content will be different let's handle it separately once we have figured out what we want to say. :)